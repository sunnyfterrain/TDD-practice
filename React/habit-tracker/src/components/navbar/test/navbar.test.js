import React from 'react';
import renderer from 'react-test-renderer';
import Navbar from '../navbar.jsx';

describe('navbar', () => {
  it('render', () => {
    // 스냅샷 테스트를 실행
    const component = renderer.create(<Navbar totalCount={4} />);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
