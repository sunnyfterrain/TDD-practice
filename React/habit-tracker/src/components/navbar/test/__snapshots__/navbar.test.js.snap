// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`navbar render 1`] = `
<nav
  className="navbar"
>
  <i
    className="navbar-logo fas fa-leaf"
  />
  <span>
    Habit Tracker
  </span>
  <span
    className="navbar-count"
  >
    4
  </span>
</nav>
`;
